Source/FullScreenPokemon.ts

```diff
@@ -9181,6 +9254,11 @@ module FullScreenPokemon {
 
             this.GamesRunner.play();
 
+            if (flying) {
+                this.shiftVert(this.player, 30);
```

Comment:
This general if block shouldn't be here but I do want to note that when shifting Things, you want to multiply by unitsize.

So, instead of 30, it should be 8 * this.unitsize (because 8 is the typical block size).